pylint_odoo/test_repo/broken_module/models/broken_model.py

```diff
@@ -245,6 +245,23 @@ def my_method1(self, variable1):
         error_msg = _('Variable not translatable: {}'.format(variable1))
         error_msg = _('Variables not translatable: {}, {variable2}'.format(
             variable1, variable2=variable2))
+
+        # string with parameters without name
+        # so you can't change the order in the translation
+        _('%s %d') % ('hello', 3)
+        _('%s %s') % ('hello', 'world')
+        _('%s %s %s') % ('hello', 'world!')
```

Comment:
This seems like a different failure. I'm not sure if normal pylint would detect it. You're using 3 paceholders with 2 args.

Possibly not interesting to implement, given in this case you should use named placeholders.

Reply:
I think that is a different failure. I removed it.